xcodeproj/internal/target_properties.bzl

```diff
@@ -163,6 +163,31 @@ def process_defines(*, cc_info, swift_info, build_settings):
 
         set_if_true(build_settings, "GCC_PREPROCESSOR_DEFINITIONS", setting)
 
+def process_exported_symbols_lists(
+        *,
+        exported_symbols_lists,
+        package_bin_dir,
+        build_settings):
+    """Logic for processing code signing flags.
+
+    Args:
+        exported_symbols_lists: A `list` of `File`s representing the
+        exported symbol lists files for the target.
+        package_bin_dir: Path to the package's bin directory.
+        build_settings: A mutable `dict` that will be updated with code signing
+            flag build settings that are processed.
+    Return:
+        The modified build settings object
+    """
+    if exported_symbols_lists and build_settings != None:
+        path_prefix = "$(BUILD_DIR)/" + package_bin_dir + "/"
```

Comment:
We shouldn't build this path. Instead we should pass a `file_path` (which will handle anything related to this file's `path`) to the generator and let it add the path to `OTHER_LDFLAGS`. I would adjust `input_files.to_dto` and `Inputs.swift` to expose this file to the generator, instead of this whole function.